packages/alert-message-react/src/AlertMessage.tsx

```diff
@@ -34,10 +43,18 @@ function alertFactory(messageType: messageTypes) {
         return (
             <div className={componentClassName} role={role}>
                 <div className="jkl-alert-message__content" data-testid="alert-message-content" style={{ ...styles }}>
-                    <div className="jkl-alert-message__icon">
+                    <div aria-hidden className="jkl-alert-message__icon">
```

Comment:
Denne ble alltid lest som "image". Vi bør sikkert vurdere å gi ikonene faktiske alt-tekster dersom de skal være med å gi meningsinnhold også for skjermlesere. Inntil det er det nok bedre å skru dem av.